### PR TITLE
configs/imxrt1050-evk, imxrt1020-evk: Add libcxx_utc for flat build

### PR DESCRIPTION
**Pull request by sangam-swami**
Add unit test case config for libcxx. Enable the CONFIG_LIBCXX_UTC.

Signed-off-by: Sangamanatha <sangam.swami@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [472e4ad6f6abe3c586bb1c0f913ba7e00c8d9df6] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [472e4ad6f6abe3c586bb1c0f913ba7e00c8d9df6] - Code Rule Check (C++) OK.

**Comment by sijupunnnoose** [comment]
How about separating 1020/1050 as 2 different patch?

**Comment by sangam-swami** [comment]
> How about separating 1020/1050 as 2 different patch?

separated into 2 commits

**Comment by seinfra** [comment]
Target : [a05115cac735403766f40b0700170e0dc5017ece] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a05115cac735403766f40b0700170e0dc5017ece] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@sangam-swami Let's enable the tc for libxx in tc defconfig. I think we don't need to make a new defconfig.

**Comment by sunghan-chang** [comment]
I already gave the same opinion in #3957 .
> @sangam-swami In my opinion, we can put c++ tcs in tc defconfig and keep helloxx only,

**Comment by seinfra** [comment]
Target : [fedb346edb31fb8cff7288e6439c6f292ba81068] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [fedb346edb31fb8cff7288e6439c6f292ba81068] - Code Rule Check (C++) OK.

**Comment by sangam-swami** [comment]
> I already gave the same opinion in #3957 .
> 
> > @sangam-swami In my opinion, we can put c++ tcs in tc defconfig and keep helloxx only,

Updated

File: build/configs/imxrt1050-evk/tc/defconfig [hunk 1]
```diff
@@ -22,6 +22,7 @@ CONFIG_FRAMEWORK_DIR="../framework"
 CONFIG_TOOLS_DIR="../tools"
 CONFIG_BUILD_FLAT=y
 # CONFIG_BUILD_PROTECTED is not set
+CONFIG_FLASH_START_ADDR=0x0
```
Comment (sunghan-chang) [inline build/configs/imxrt1050-evk/tc/defconfig]: CONFIG_FLASH_START_ADDR=0x60000000

File: build/configs/imxrt1050-evk/tc/defconfig [hunk 2]
```diff
@@ -937,12 +987,15 @@ CONFIG_TC_FS_PROCFS=y
 # CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC is not set
 # CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_ITC is not set
 # CONFIG_EXAMPLES_TESTCASE_SECURITY_FRAMEWORK_UTC is not set
+# CONFIG_LIBCXX_UTC is not set
```
Comment (sunghan-chang) [inline build/configs/imxrt1050-evk/tc/defconfig]: Isn't it the main topic of this commit? This is not enabled.

Comment (sangam-swami) [reply build/configs/imxrt1050-evk/tc/defconfig]: I can enable these test cases by default. I thought like some other test cases, can be enabled during specific tests, might increase binary size.

File: build/configs/imxrt1050-evk/tc/defconfig (hunk 1 above)
Comment (sangam-swami) [reply build/configs/imxrt1050-evk/tc/defconfig]: updated

**Comment by seinfra** [comment]
Target : [8736d6655f92f43f3a3cb135da757da2dea653db] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8736d6655f92f43f3a3cb135da757da2dea653db] - Code Rule Check (C++) OK.

**Pull request merged**